includes/push-ui.php

```diff
@@ -498,13 +537,16 @@ function menu_content() {
 								 */
 								$as_draft = apply_filters( 'dt_allow_as_draft_distribute', $as_draft, $connection, $post );
 							?>
-							<button class="syndicate-button"><?php esc_html_e( 'Distribute', 'distributor' ); ?></button> <?php if ( $as_draft ) : ?><label class="as-draft" for="dt-as-draft"><input type="checkbox" id="dt-as-draft" checked> <?php esc_html_e( 'As draft', 'distributor' ); ?></label><?php endif; ?>
+							<button class="syndicate-button"><?php esc_html_e( 'Distribute', 'distributor' ); ?></button>
+							<?php if ( $as_draft ) : ?>
+								<label class="as-draft" for="dt-as-draft"><input type="checkbox" id="dt-as-draft" checked> <?php esc_html_e( 'As draft', 'distributor' ); ?></label>
+							<?php endif; ?>
 						</div>
 					</div>
 
 					<div class="messages">
 						<div class="dt-success">
-							<?php esc_html_e( 'Post successfully distributed.', 'distributor' ); ?>
+							<?php echo esc_html( apply_filters( 'dt_successfully_distributed_message', esc_html__( 'Post successfully distributed.', 'distributor' ) ) ); ?>
```

Comment:
Is this a new filter? if so, needs documenting.

The inner `esc_html__` can change to `__` - escaping should happen late, right before output.

Reply:
Changed accordingly.